pyproject.toml

```diff
@@ -28,6 +28,7 @@ urllib3 = { version = ">=1.26.17,<2.0.0" }
 requests = { version = ">=2.1.0,<3.0.0" }
 openepd = { version = ">=3.1.0" }
 pytz = { version = ">=2023.3" }
+beautifulsoup4 = { version = ">=4.10.0" }
```

Comment:
Why do we need beatutiful soap if we already have and use mechanical soup?
See `cqd_etl.ingestion.common.scrapping.impl.msoap.BaseMechanicalSoapSpider` and it's child classes for reference.